src/game/behaviors/mr_i.inc.c

```diff
@@ -76,7 +76,7 @@ void bhv_mr_i_iris_loop(void) {
         obj_build_transform_from_pos_and_angle(o, O_PARENT_RELATIVE_POS_INDEX, O_MOVE_ANGLE_INDEX);
         obj_translate_local(o, O_POS_INDEX, O_PARENT_RELATIVE_POS_INDEX);
         o->oFaceAnglePitch = o->oMoveAnglePitch;
-        o->oGraphYOffset = o->header.gfx.scale[1] * 100.0f;
+        o->oGraphYOffset = o->parentObj->oBehParams2ndByte ? 200.0f : 100.0f;
```

Comment:
The Y offset for Mr. I was changed in HackerSM64 so that the billboard origin is at the texture's actual center instead of at the bottom, which fixed a few minor vanilla visual bugs that were more obvious with HackerSM64's camera changes. The Y offset is also scaled based on the bparam so that the larger Mr. I in BBH works too. The issue here was that the iris's Y offset was being scaled along with the actual scale during the death animation. A better fix for this would actually just be by copying the parent object's oGraphYOffset.